src/DayPicker.tsx

```diff
@@ -380,10 +380,32 @@ export function DayPicker(props: DayPickerProps) {
                           value={dateLib.getYear(calendarMonth.date)}
                         />
                       ) : (
-                        <span role="status" aria-live="polite">
+                        <span>
                           {formatYearDropdown(calendarMonth.date, dateLib)}
                         </span>
                       )}
+                      <span
```

Comment:
@gpbl should we extract this to a separate component to keep the `DayPicker` body shorter?

Reply:
Before, we had a “visuallyHidden” CSS class for this purpose. However, I prefer the inline style in this case for backward compatibility. 

I agree that we should extract parts of this component into subcomponents in the near future!

Reply:
Makes sense 👍 
Wanted to check what would be best for now.
We can wait for another place using the same pattern to extract to a separate component.